src/main/java/org/galatea/starter/domain/rpsy/IexHistoricalPricesKey.java

```diff
@@ -0,0 +1,18 @@
+package org.galatea.starter.domain.rpsy;
+
+import java.io.Serializable;
+import java.util.Date;
+
+public class IexHistoricalPricesKey implements Serializable {
```

Comment:
Because this is being used as a composite key, it required the key to implement serializable. I'm not sure why, internally, this is the case. 

I'll use @Data instead of explicit constructors.